code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -150,20 +150,21 @@
 	item_state_slots = list(slot_r_hand_str = "medical_voidsuit_bio", slot_l_hand_str = "medical_voidsuit_bio")
 	armor = list(melee = 45, bullet = 5, laser = 20, energy = 5, bomb = 15, bio = 100, rad = 75)
 
-//Medical Surplus Voidsuit
-/obj/item/clothing/head/helmet/space/void/medical/alt
-	name = "streamlined medical voidsuit helmet"
-	desc = "A trendy, lightly radiation-shielded voidsuit helmet trimmed in a fetching green."
-	icon_state = "rig0-medicalalt"
-	armor = list(melee = 30, bullet = 5, laser = 10,energy = 5, bomb = 5, bio = 100, rad = 60)
-	light_overlay = "helmet_light_dual_green"
-
-/obj/item/clothing/suit/space/void/medical/alt
-	icon_state = "rig-medicalalt"
-	name = "streamlined medical voidsuit"
-	desc = "A more recent model of Vey-Med voidsuit, featuring the latest in radiation shielding technology, without sacrificing comfort or style."
-	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,/obj/item/weapon/storage/firstaid,/obj/item/device/healthanalyzer,/obj/item/stack/medical)
-	armor = list(melee = 30, bullet = 5, laser = 10,energy = 5, bomb = 5, bio = 100, rad = 60)
+//Medical Streamlined Voidsuit
+	/obj/item/clothing/head/helmet/space/void/medical/alt
+		name = "streamlined medical voidsuit helmet"
+		desc = "A trendy, lightly radiation-shielded voidsuit helmet trimmed in a fetching green."
+		icon_state = "rig0-medicalalt"
+		armor = list(melee = 30, bullet = 5, laser = 20,energy = 5, bomb = 25, bio = 100, rad = 80)
+		light_overlay = "helmet_light_dual_green"
+
+	/obj/item/clothing/suit/space/void/medical/alt
+		icon_state = "rig-medicalalt"
+		name = "streamlined medical voidsuit"
+		desc = "A more recent model of Vey-Med voidsuit, featuring the latest in radiation shielding technology, without sacrificing comfort or style."
+		allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,/obj/item/weapon/storage/firstaid,/obj/item/device/healthanalyzer,/obj/item/stack/medical,/obj/item/weapon/reagent_containers/hypospray,/obj/item/weapon/reagent_containers/glass,/obj/item/weapon/storage/pill_bottle,/obj/item/roller,/obj/item/bodybag/cryobag)
```

Comment:
Do me a favor and compare this to the list on space/void/medical please?

Reply:
Alright, got the comparison. Let's comment out this line for the time being. I don't know why it's so expanded.